pkgs/by-name/li/limine/package.nix

```diff
@@ -0,0 +1,58 @@
+# Builds limine with all available features.
+
+{
+  # Helpers
+  stdenv
+, fetchurl
+, lib
+, # Dependencies
+  llvmPackages
+, mtools
+, nasm
+}:
+
+let
+  version = "7.1.0";
+in
+# The output of the derivation is a tool to create bootable images using Limine
+# as bootloader for various platforms and corresponding binary and helper files.
+stdenv.mkDerivation {
+  inherit version;
+  pname = "limine";
+  # We don't use the Git source but the release tarball, as the source has a
+  # `./bootstrap` script performing network access to download resources.
+  # Packaging that in Nix is very cumbersome.
+  src = fetchurl {
+    url = "https://github.com/limine-bootloader/limine/releases/download/v${version}/limine-${version}.tar.gz";
+    sha256 = "sha256-oaQA461MGVuG7aJri2HyYb6Pu8BXOWTF3Wq/SputQSw=";
+  };
+
+  nativeBuildInputs = [
+    llvmPackages.bintools
+    # gcc is used for the host tool, while clang is used for the bootloader.
+    llvmPackages.clang
+    llvmPackages.lld
+    mtools
+    nasm
+  ];
+
+  configureFlags = [ "--enable-all" ];
+
+  enableParallelBuilding = true;
+
+  outputs = [ "out" "doc" "dev" "man" ];
+
+  meta = with lib; {
+    homepage = "https://limine-bootloader.org/";
+    description = "Limine Bootloader and Tools";
+    # Caution. Some submodules have different licenses.
+    license = licenses.bsd2;
```

Comment:
The maintainer said it is okay to only expose this single license.

Reply:
Well, we didn't really say that, we are not lawyers... If it's better to be explicit and list the licenses of all submodules so be it. I guess it would be worth looking for similar precedents in nixpkgs to see if all submodule licenses get exposed or not.